packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -629,6 +630,51 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 		}
 	}
 
+	public get disposed() {
+		if (!(this._disposed || this.socket.connected)) {
+			this.logger.sendTelemetryEvent({
+				eventName: "ConnectionNotYetDisposed",
+				driverVersion: pkgVersion,
+				details: JSON.stringify({
+					...this.getConnectionDetailsProps(),
+				}),
+			});
+		}
+		return this._disposed;
+	}
+
+	private get connected(): boolean {
+		return !this.disposed && this.socket.connected;
+	}
+
+	protected emitMessages(type: string, messages: IDocumentMessage[][]) {
+		// Only submit the op/signals if we are connected.
+		if (this.connected) {
+			this.socket.emit(type, this.clientId, messages);
+		}
+	}
+
+	protected submitCore(type: string, messages: IDocumentMessage[]) {
```

Comment:
I do not see a value in this function - I'd remove it and make callers call emitMessages() directly